tests/test_publication.py

```diff
@@ -1,87 +1,62 @@
 from unittest import mock
 
+import pytest
 import rdflib
 from rdflib.namespace import RDF
 
 from nanopub import namespaces, Publication
+from nanopub.definitions import DUMMY_NANOPUB_URI
 
-TEST_ASSERTION = (namespaces.AUTHOR.DrBob, namespaces.HYCL.claims, rdflib.Literal('This is a test'))
 TEST_ORCID_ID = 'https://orcid.org/0000-0000-0000-0000'
 
 
 class TestPublication:
```

Comment:
I've removed a lot of redundant tests, and made each test test a specific thing.

Reply:
👍